src/routes/detail/[slug]/+page.svelte

```diff
@@ -16,7 +16,7 @@
 	<!-- Image -->
 	<div class="">
 		{#if data.content?.Bild}
-			<a href="{base}#">
+			<a href="{base}/#">
```

Comment:
```suggestion
			<a href="{base}/">
```
We don't need the hashtag anymore here.